pkg/db/db_local/start_services.go

```diff
@@ -156,13 +156,23 @@ func postServiceStart(ctx context.Context, res *StartResult) error {
 	if err := setupInternal(ctx, conn); err != nil {
 		return err
 	}
-	// ensure connection stat etable contains entries for all connections in connection config
+	// ensure connection state table contains entries for all connections in connection config
 	// (this is to allow for the race condition between polling connection state and calling refresh connections,
 	// which does not update the connection_state with added connections until it has built the ConnectionUpdates
 	if err := initializeConnectionStateTable(ctx, conn); err != nil {
 		return err
 	}
 
+	// initialize the server settings table
```

Comment:
// initialize the server settings table
	// this table contains that configuration for this service instance
	// (we don't support file watching on service config file so the settings 
	//  remain valid for the lifetime of this instance)